content/en/docs/Reference/file-based-catalogs.md

```diff
@@ -304,6 +304,49 @@ properties:
     kind: Bar
 ```
 
+#### `olm.deprecations`
+
+Operator authors can use the file-based catalog (FBC) deprecation schema to provide relevant information for support and upgrades for users running operators. The deprecation schema follows the Meta FBC Schema conventions. 
+
+You can use the `olm.deprecations` schema to deprecate packages, bundles, and channels. This schema consists of a list of scoped references to catalog content, with a descriptive message string that provides more information to the user. The deprecation schema is an optional feature.
+
+A valid deprecation schema meets the following criteria:
+- There must be only one schema per package
+- The message must be a non-zero length
+- The package name must exist in the bundle
+  
+The deprecation feature does not consider overlapping deprecation (package vs channel vs bundle).
+
+|              | `olm.package`                   | `olm.channel`        | `olm.bundle`          |
+|--------------|---------------------------------|----------------------|-----------------------|
+| Scope        | Entire Package                  | Single Channel       | Single Bundle Version |
+| Requirements | `name` must be empty | `name`  is mandatory | `name`  is mandatory  |
+
+For the entry type `olm.package` the `name` will be inherited from the parent `olm.deprecations`
```

Comment:
Not needed if https://github.com/operator-framework/olm-docs/pull/323/files#r1480491816 accepted.

Reply:
Do we need to have this both [here](https://github.com/operator-framework/olm-docs/pull/323/files#diff-afc86fd86db68fce1c3f31d0b608dc396467c3f349adb2f992427635420b776eR323) and here?